catalogue/webapp/pages/worksv2.js

```diff
@@ -74,7 +74,7 @@ export const Works = ({
         imageUrl={null}
         imageAltText={null}>
         <ErrorPage
-          errorStatus={works.httpStatus}
+          statusCode={works.httpStatus}
         />
       </PageLayout>
```

Comment:
As above, <PageLayout> is now included in <ErrorPage> so we can remove it from here